### PR TITLE
New Bosch Parking Lot Sensor

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/bosch/parkingLotSensor/schema.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "$id": "https://akenza.io/bosch/parkingLotSensor.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Bosch Parking Lot Sensor",
+  "type": "object",
+  "properties": {
+    "occupancy": {
+      "description": "Space occupancy. 0 = Unoccupied / 1 = Occupied.",
+      "type": "integer",
```
Comment (mariusgiger) [inline types/bosch/parkingLotSensor/schema.json:9]: better use a boolean here?

File: types/bosch/parkingLotSensor/meta.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Bosch Parking Lot Sensor",
+  "version": "1.0.0",
+  "manufacturer": "Bosch",
+  "url": "https://www.bosch-connectivity.com/products/connected-mobility/parking-lot-sensor/",
+  "description": "Parking Lot Sensor to measure the occupancy of the parking lot",
```
Comment (mariusgiger) [inline types/bosch/parkingLotSensor/meta.json:6]: change to "a parking lot"

File: types/bosch/parkingLotSensor/uplink.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  if (port == 3) {
+    var reset_dict = {
+      0x01: "Watchdog reset",
+      0x02: "Power On Reset",
+      0x03: "System Request Reset",
+      0x04: "Other Resets"
+    };
+    data.debug = "0x" + payload.substring(0, 24).toUpperCase();
+    data.fw_version =
```
Comment (mariusgiger) [inline types/bosch/parkingLotSensor/uplink.js:19]: add these properties to the schema

File: types/bosch/parkingLotSensor/uplink.spec.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+before(function (done) {
+  fs.readFile(__dirname + "/schema.json", "utf8", function (err, fileContents) {
+    if (err) throw err;
+    schema = JSON.parse(fileContents);
+    done();
+  });
+});
+
+
+describe("Bosch Parking Lot Sensor Uplink", function () {
```
Comment (mariusgiger) [inline types/bosch/parkingLotSensor/uplink.spec.js:27]: add a test with additional information

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, some improvement porposals

File: types/bosch/parkingLotSensor/schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/bosch/parkingLotSensor/schema.json:9]: We always use int so boolean would be a little bit out of the standard

**Pull request merged**